src/applications/healthcare/questionnaire/api/appointment-data.json

```diff
@@ -1,57 +1,42 @@
 {
-  "id": "ABC-123",
-  "startDate": "2020-08-26T18:16:04.504Z",
-  "clinicId": "clinic-id",
-  "clinicFriendlyName": "My Cool Clinic",
-  "clinicPhone": "555-867-5309",
-  "clinicPhoneExtension": "",
-  "facilityId": "facility-id",
-  "sta6aid": "",
-  "stationName": "",
-  "patientIcn": "patient-icn-from-json",
-  "communityCare": true,
-  "vvsAppointments": [],
-  "vdsAppointments": [
-    {
-      "id": "string",
-      "appointmentLength": "string",
-      "appointmentTime": "2020-12-02T13:15:00.000Z",
-      "status": "string",
-      "clinic": {
-        "id": "string",
-        "name": "string",
-        "department": "string",
-        "service": "string",
-        "specialty": "string",
-        "stopCode": "string",
-        "facility": {
-          "siteCode": "string",
-          "name": "string",
-          "displayName": "Douglas VA Medical Center",
-          "regionId": "string",
-          "lastEventTimestamp": "string",
-          "lastEventReason": "string",
-          "uid": "string",
-          "pid": "string",
-          "type": "string",
-          "facilityCode": "string",
-          "state": "string",
-          "city": "string",
-          "address": "string",
-          "parentSiteCode": "string"
-        },
-        "askForCheckIn": false,
-        "facilityCode": "string",
-        "dssId": "string",
-        "inactivate": "string",
-        "reactivate": "string"
-      },
-      "patientId": "string",
-      "bookingNotes": "Follow-up/Routine: Pain in right knee after surgery",
-      "labDateTime": "2020-08-26T18:16:04.504Z",
-      "ekgDateTime": "2020-08-26T18:16:04.504Z",
-      "xrayDateTime": "2020-08-26T18:16:04.504Z"
-    }
-  ],
-  "ccsAppointments": []
+	"data": {
+		"id": "195bc02c0518870fc6b1e302cfc326b0",
+		"type": "va_appointments",
+		"attributes": {
+			"startDate": "2020-08-26T15:00:00Z",
+			"sta6aid": "983",
+			"clinicId": "848",
+			"clinicFriendlyName": "CHY PC VAR2",
+			"facilityId": "983",
+			"communityCare": false,
+			"patientIcn": "1013124304V115761",
+			"vdsAppointments": [{
+				"bookingNotes": "Follow-up/Routine: testing reason for visit field availability",
+				"appointmentLength": "20",
+				"id": "848;20200826.090000",
+				"appointmentTime": "2020-08-26T15:00:00Z",
+				"clinic": {
+					"name": "CHY PC VAR2",
+					"askForCheckIn": false,
+					"facilityCode": "983",
+					"facility": {
+						"displayName": "VDS Facility Display Name"
+					},
+					"stopCode": "323"
+				},
+				"type": "REGULAR",
+				"currentStatus": "NO ACTION TAKEN/TODAY"
+			}],
+			"vvsAppointments": [{
+				"id": "abc",
+				"status": "defgh",
+				"patients": [{
+					"name": "Foo Bar"
+				}],
+				"providers": [{
+					"name": "The Provider"
+				}]
+			}]
+		}
+	}
```

Comment:
Is this still needed? Are we pulling the mock appointment from the vets-api yet?

Reply:
Yes, for e2e and unit testing environments.

Reply:
To clarify, yes its needed for the testing, and yes we are pulling from the vets-api